src/model/account/PublicAccount.ts

```diff
@@ -53,6 +54,43 @@ export class PublicAccount {
         return new PublicAccount(publicKey, address);
     }
 
+    /**
+     * Verify a signature.
+     *
+     * @param {string} data - The data to verify.
+     * @param {string} signature - The signature to verify.
+     *
+     * @return {boolean}  - True if the signature is valid, false otherwise.
+     */
+    public verifySignature(data: string, signature: string): boolean {
+        if (!data || !signature) {
```

Comment:
is it possible there is a case where the signed data is ""? (no data)

Reply:
Yes, does it make sense? Or should we thrown an exception?